src/utils/standardLabels.ts

```diff
@@ -0,0 +1,9 @@
+export const standardLabelsBarn: any = {
+  fnr: 'Fødselsnummer',
+  personnummer: 'Personnummer',
+  navn: 'Navn',
+  alder: 'Alder',
+  fødselsdato: 'Fødselsdato',
+  harSammeAdresse: 'Har barnet samme adresse som deg?',
+  født: 'Er barnet født?',
+};
```

Comment:
Jeg lærte nylig at `utils` brukes om generelle metoder som kan brukes overalt i kodebasen, mens `helpers` er for litt mer spesifikke hjelpefunksjoner eller konstanter - noe i dette tilfellet jeg mener er en slik helpers funksjon 😄  

I min branch har jeg laget en `src/helpers` - mappe jeg bruker til "tomme" feltobjekter etc. Kanskje denne også kunne vært flyttet inn hit?

Reply:
Ah, nice, da la jeg denne inn i helpers 😊